react.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
   rules: {
     'react-hooks/exhaustive-deps': 'warn',
     'react-hooks/rules-of-hooks': 'error',
+    'react/display-name': 'off',
```

Comment:
This is the default behavior.

You should be able to add an example case to demonstrate that this rule is not being enforced already:
```js
import React from 'react';

export default class extends React.Component {
  render() {
    return <div>no display name</div>;
  }
}
```

Reply:
OK will do that thanks. I just spotted this old branch and we also have the rule set on our side but maybe that is obsolete by now, will add a test tomorrow 👍

Reply:
So I spot a couple of warning that are thrown in our code:

![Screenshot 2020-11-10 at 09 29 47](https://user-images.githubusercontent.com/1742115/98656169-abac3180-2340-11eb-83ca-321590762b99.png)

![Screenshot 2020-11-10 at 09 31 28](https://user-images.githubusercontent.com/1742115/98656200-b5359980-2340-11eb-82f3-dcb36fbe0306.png)
![Screenshot 2020-11-10 at 09 31 09](https://user-images.githubusercontent.com/1742115/98656209-b666c680-2340-11eb-8726-b3aa6210b4ea.png)
![Screenshot 2020-11-10 at 09 31 02](https://user-images.githubusercontent.com/1742115/98656213-b6ff5d00-2340-11eb-9387-c4ddcaba50d0.png)

Will come up with a good example.

Reply:
Upon researching @bartvde found out that these are false positives https://github.com/yannickcr/eslint-plugin-react/issues/2269#issuecomment-491567533 so we fixed this on our side: https://hello.planet.com/code/product/om-web/-/merge_requests/1503 making this PR obsolete 🎉